packages/core/scripts/collateral-umip/1_Propose.js

```diff
@@ -89,9 +93,26 @@ async function runExport() {
     transactionList = [...transactionList, ...transactionsToAdd];
   }
 
-  // Send the proposal
   const governor = await Governor.deployed();
-  await governor.propose(transactionList, { from: proposerWallet, gas: 2000000 });
+  console.log(`Sending to governor @ ${governor.address}`);
```

Comment:
Helpful information for knowing which Voting the Governor will be proposing requests to